src/Sleepycat.xml

```diff
@@ -0,0 +1,115 @@
+<!-- REVIEW -->
```

Comment:
The first line appears suspicious, please check.  Once the first line is approved this license is approved.